ops/check-changed/main.py

```diff
@@ -122,6 +131,15 @@ def match_path(path, patterns):
             return True
     return False
 
+def extract_pr_number(branch_name):
+    # Merge queue branches are named: gh-readonly-queue/{base_branch}/pr-{number}-{sha}
+    match = re.search(r'/pr-(\d+)-', branch_name)
```

Comment:
This can return an incorrect PR number if the `base_branch` name also matches the regex, but that seems unlikely, so figured I'd keep the regex simple